ui/store/actions.ts

```diff
@@ -4645,6 +4648,17 @@ export function setSnapsInstallPrivacyWarningShownStatus(shown: boolean) {
 }
 ///: END:ONLY_INCLUDE_IN
 
+///: BEGIN:ONLY_INCLUDE_IN(build-flask)
+export function trackInsightSnapUsage(snapId: string) {
+  return async () => {
+    await submitRequestToBackground('_trackSnapExportUsage', [
+      snapId,
+      HandlerType.OnTransaction,
```

Comment:
Should this be a separate event instead? It's kind of misleading to track the export differently 🤔